shared/functions.lua

```diff
@@ -10,57 +10,68 @@ function public.getIsCloseToCoords(coord1, coord2, distance)
     return #(coord1 - coord2) < distance
 end
 
-local function getIsVehicleOnList(vehicle, list)
-    local vehicleHash = GetEntityModel(vehicle)
+local function getIsOnList(item, list)
     for i = 1, #list do
-        if vehicleHash == joaat(list[i]) then
+        if item == list[i] then
             return true
         end
     end
 end
```

Comment:
Replace with https://overextended.dev/ox_lib/Modules/Table/Shared#libtablecontains so this function can be deleted.

Reply:
I'm not sure it's the best option when we care about performance
https://github.com/overextended/ox_lib/blob/7ecd7dc21fe5f554a0bb49f058063395d9d03402/imports/table/shared.lua#L7-L30

Reply:
Because of the pairs vs iterative? Fine to keep this then.

Reply:
exactly